services/efs-guardian/tests/unit/test_core_settings.py

```diff
@@ -0,0 +1,12 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+
+from pytest_simcore.helpers.utils_envs import EnvVarsDict
+from simcore_service_efs_guardian.core.settings import ApplicationSettings
+
+
+def test_settings(app_environment: EnvVarsDict):
```

Comment:
same story here. Moreover, if you do it using https://github.com/ITISFoundation/osparc-simcore/blob/master/services/api-server/tests/unit/test_core_settings.py#L15-L40 you can test your settings against any of the repo.config files ... which is very handy while developing